shared/domain/src/main/java/com/ivy/domain/features/IvyFeatures.kt

```diff
@@ -9,11 +9,17 @@ class IvyFeatures @Inject constructor() : Features {
         key = "sort_categories_alphabetically",
         name = "Sort Categories Alphabetically",
         description = "Sort income and expenses" +
-        " categories alphabetically"
+                " categories alphabetically"
+    )
+    override val showCompactAccounts = BoolFeature(
+        key = "show_compact_accounts",
+        name = "Show Compact Accounts",
+        description = "Loads accounts in Dense Account mode"
```

Comment:
Enables more compact and dense UI for the "Accounts" tab

Reply:
Done